### ISSUE TITLE
Billing alert practices - what to do / figure out what to do when an alert fires?

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/infrastructure#3622]
I [wrote the following in `#support-freshdesk` on slack](https://2i2c.slack.com/archives/C028WU9PFBN/p1705485641603839), and @jmunroe messaged a community representative to confirm they were aware about the situation, and that was the resolution to this billing alert.

> Is there a procedure on handling the budget billing alerts fired to [support@2i2c.org](mailto:support@2i2c.org) ? I'm not sure what to do with https://2i2c.freshdesk.com/a/tickets/1246 that sais 90% of a yearly LEAP budget has been consumed with a bit more than 2 months remaining.

How should we as support triager act when receiving an alert like this?

### Loose thoughts

- If a billing alert is monthly, its perhaps worth treating a bit differently from a yearly billing alert for example
- I don't know, but maybe engineering setup billing alerts to catch unexpected expenditure, and no community champion is involved to consider this etc?
- If we get an billing alert of "90% of the budget" alert after 20% of the alert period has passed, then its a bigger deal than if its a "90% of the budget" alert after 90% of the alert period has passed.

**Comment by jmunroe** [comment]
I had some months ago, tried to set up a billing alert for LEAP that would trigger if any *non-credit* money was spent. That alert never work as I intended so I have just deleted it. 

There are still these LEAP related budget alerts set up:

<img width="787" alt="image" src="https://github.com/2i2c-org/infrastructure/assets/6181563/40fb48e2-76b9-4eb1-9824-4a24880b107f">

I agree that it is *not* clear what a support triager should do when seeing this alerts.  There may also be alerts from other projects as well.

Is there a way we can considered these not to be support request and someone be an input into a standard operating procedures around cloud billing?  Is 'budget alerts' already included in the quarterly goal around cloud billing?
